RecoTracker/LSTCore/standalone/bin/lst.cc

```diff
@@ -383,6 +400,10 @@ void run_lst() {
     if (not goodEvent())
       continue;
 
+    // Backwards compatibility
+    const auto trk_ph2_clustSize =
+        hasClustSize ? trk.getVUS("ph2_clustSize") : std::vector<uint16_t>(trk.getVF("ph2_x").size());
```

Comment:
do you want to support the interim `size_t`-sized variant as well?

Reply:
Honestly, I don't want to 🤣 but I'm willing to if you think it's appropriate